providers/base/units/ethernet/test-plan.pxu

```diff
@@ -17,6 +17,16 @@ include:
 bootstrap_include:
     device
 
+id: ethernet-wake-on-lan-cert-manual
+unit: test plan
+_name: Ethernet wake on lan tests (manual)
+_description: Ethernet wake on lan tests (manual)
+include:
+    ethernet/wol_S5_.*
+    ethernet/wol_S3_.*
```

Comment:
Now that we can use template IDs in test plan definition, it's better to use them (less regular expressions :))

```suggestion
    ethernet/wol_S5_interface
    ethernet/wol_S3_interface
```